src/Model/Order/Order.php

```diff
@@ -64,16 +83,46 @@ public function getContactPerson(): NonNullContactPerson
         return $this->contactPerson;
     }
 
-    public function getHasPaid(): NonNullOrderIsPaid
+    public function getPaymentStatus(): NonNullPaymentStatus
     {
-        return $this->hasPaid;
+        return $this->paymentStatus;
     }
 
     public function getAmount(): NonNullOrderAmount
     {
         return $this->amount;
     }
 
+    public function getDatePaid(): DatePaid
+    {
+        return $this->datePaid;
+    }
+
+    public function getPaymentId(): NonNullEntityId
+    {
+        return $this->paymentID;
+    }
+
+    public function getPayeeName(): NonNullPayeeName
+    {
+        return $this->payeeName;
+    }
+
+    public function getPaymentDate(): NonNullPaymentDate
+    {
+        return $this->paymentDate;
+    }
+
+    public function getPaymentAmount(): NonNullPaymentAmount
```

Comment:
If this is for tracking the total amount paid, I think we were going to have a method to calculate, rather than a property on the aggregate

Reply:
This has been now changed.